lostmediafinder/finder.py

```diff
@@ -337,6 +337,32 @@ async def _run(cls, id, session: aiohttp.ClientSession):
             rawraw=raw, metaonly=False, comments=False, available=available
         )
 
+class removededm(YouTubeService):
+    """
+    Queries the removed.edm database for the video you requested.
+    """
+    name = methods["removededm"]["title"]
+    configId = "removededm"
+
+    @classmethod
+    async def _run(cls, id, session: aiohttp.ClientSession) -> T:
+        ismeta = False
+        lien = f"https://removededm.com/wiki/File:{id}.mp4" or f"https://removededm.com/wiki/File:{id}.webm"
+        async with session.head(lien, allow_redirects=False, timeout=15) as response:
+            redirect = response.headers.get("location")
+            archived = bool(redirect) # if there's a redirect, it's archived
+        if not archived:
+            lien = f"https://removededm.com/wiki/{id}"
```

Comment:
Is it possible for the site to have pages for videos that don't have metadata?

Reply:
Generally all of the pages that exist have some sort of metadata (upload dates, video titles, descriptions, thumbnails, etc.) so it would be a very rare thing (if not impossible) to have a page without any metadata.